pkg/reconcile/pipeline/builder/pipeline_integration_test.go

```diff
@@ -152,7 +153,7 @@ var _ = Describe("Default Pipeline", func() {
 		updatedApp := &appsv1.Deployment{}
 		err = runtime.DefaultUnstructuredConverter.FromUnstructured(u.Object, updatedApp)
 		Expect(err).NotTo(HaveOccurred())
-		Expect(updatedApp.Spec.Template.Spec.Containers[0].EnvFrom[0].SecretRef.Name).To(Equal(updatedSB.Status.Secret))
+		//Expect(updatedApp.Spec.Template.Spec.Containers[0].EnvFrom[0].SecretRef.Name).To(Equal(updatedSB.Status.Secret))
```

Comment:
if not needed, please remove.

Reply:
Reverted. 1765cebdf501d466932aaa6708621ff5d3a3e087
(I commented it earlier as it was failing)